xml/obs_scm_bridge.xml

```diff
@@ -12,31 +12,34 @@
          xmlns:ns5="http://www.w3.org/2000/svg"
          xmlns:ns4="http://www.w3.org/1999/xhtml"
          xmlns:ns="http://docbook.org/ns/docbook">
-  <title>SCM Bridge [EXPERIMENTAL]</title>
+  <title>SCM Bridge</title>
 
   <sect1 xml:id="sec.obs.obs_scm_bridge">
     <title>SCM Bridge</title>
 
     <sect2 xml:id="sec.obs.obs_scm_bridge.setup.introduction">
       <title>Introduction</title>
 
-      <para>The SCM bridge allows to setup single packages or entire projects in any trusted SCM repositories. 
+      <para>The SCM bridge allows to setup single package or entire project sources to be stored and
+            maintained in any trusted SCM repositories. 
             However, the only supported SCM is currently git.</para>
 
-      <para>This allows to manage the entire package sources in an external SCM repository. While the classic
-      OBS services for scms (like obs_scm or tar_scm) are using the workflows inside of OBS, these workflows
-      have to happen inside of the SCM provider instance. OBS will just verify the authenticity of that resource in
-      future.</para>
+      <para>This allows to run the entire packaging source update in an external SCM repository using the
+      native tooling. It is recommended to have this SCM server on the same trust level as the OBS
+      instance. However, any SCM server reachable via network could be used.</para>
 
-      <para>osc can be used to checkout sources. This will create a scm repository, so any modification needs to be
+      <para>osc can still be used to checkout sources. This will create a scm repository, so any modification needs to be
       done using the native scm tooling (eg. git). osc can still be used for local building and checking server
       side build results.</para>
 
-      <para>Be aware that any source mechanics like link or branch of OBS are not working when using the bridging.
-      This means also that workflows are not working anymore or are limited. For example a submit request would
-      still work when having scmsync on package level, but the user would manually specify the target always.
-      It won't work at all when the entire project is managed via git. This briding mechanic is designed to follow
-      the configured git provider, so this instance need to provide any required workflow.</para>
+      <para>Be aware that any source modification mechanics, submit requests or link behaviour may work
+      different or not all. For example you can not use automatic package source merging via _link files anymore.
+      This means also that workflows inside of OBS are not working anymore or are limited. A submit or release
+      request would still work with a scmsync source, but the user would manually need to specify the target always.
+      Functionality for project managed resources is even more limited.</para>
```

Comment:
Can you give an example or two of these "project managed resources" for readers who aren't familiar with this term?

Reply:
I have reworeded this now.